internal/mode/static/nginx/config/servers.go

```diff
@@ -392,12 +431,12 @@ func createRewritesValForRewriteFilter(filter *dataplane.HTTPURLRewriteFilter, p
 	return rewrites
 }
 
-// httpMatch is an internal representation of an HTTPRouteMatch.
-// This struct is marshaled into a string and stored as a variable in the nginx location block for the route's path.
-// The NJS httpmatches module will look up this variable on the request object and compare the request against the
-// Method, Headers, and QueryParams contained in httpMatch.
+// routeMatch is an internal representation of an HTTPRouteMatch.
```

Comment:
could you replace more all occurrences of httpMatch with routeMatch in the description?